tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Utils/IntegrationTestDatabase.cs

```diff
@@ -14,6 +14,7 @@ public static async Task ResetAsync(IConfiguration config)
             await databaseConnection.ExecuteAsync("GRANT CONNECT TO [NHSD-ORDAPI];");
             await databaseConnection.ExecuteAsync("ALTER ROLE db_datareader ADD MEMBER [NHSD-ORDAPI];");
             await databaseConnection.ExecuteAsync("ALTER ROLE db_datawriter ADD MEMBER [NHSD-ORDAPI];");
```

Comment:
I think you just need to put it on the serviceRecipients foreign key

Reply:
I've put it on the order and service recipient foreign keys. As either could be deleted that order items are associated with. @bjss-chris was on hand to discuss this one.

Reply:
I've had to leave the deletes in & remove the on cascade delete statements as it was creating a circular chain so the dacpac would not deploy.